src/SFA.DAS.EmployerIncentives.Api/Startup.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using SFA.DAS.EmployerIncentives.Api.AppStart;
+using SFA.DAS.EmployerIncentives.Api.Authentication;
+using SFA.DAS.EmployerIncentives.Api.Authorization;
+using SFA.DAS.EmployerIncentives.Api.Configuration;
+using SFA.DAS.EmployerIncentives.Api.ErrorHandler;
+using SFA.DAS.EmployerIncentives.Api.HealthChecks;
+using IHostingEnvironment = Microsoft.AspNetCore.Hosting.IHostingEnvironment;
+
+namespace SFA.DAS.EmployerIncentives.Api
+{
+    public class Startup
+    {
+        private readonly IWebHostEnvironment _env;
+        private readonly IConfiguration _configuration;
+
+        public Startup(IConfiguration configuration, IWebHostEnvironment env)
+        { _env = env;
+            _configuration = configuration;
+
+            var config = new ConfigurationBuilder()
+                .AddConfiguration(configuration);
+
+            if (_env.IsDevelopment())
+            {
+                config.AddJsonFile($"appsettings.development.json", optional: true);
+            }
+
+            if (_env.IsEnvironment("AcceptanceTests"))
+            {
+                config.AddJsonFile($"appsettings.acceptancetests.json", optional: false);
+            }
+
```

Comment:
Having code specific for tests makes me sad on the inside.

Reply:
I know. But for the FAKE stubs to work we need to override teh default config setting in AzureStorage. I guess the file will get loaded in an acceptancetest environment, but we need to place it here so it is loaded last.

Reply:
Will look at moving these into memory variables for acceptance tests